src/components/fileStorage/uc/deleteUserData.uc.js

```diff
@@ -0,0 +1,40 @@
+const logger = require('../../../logger');
+
+const repo = require('../repo/files.repo');
+
+/**
+ * Delete file connections for files shared with user
+ * @param {BSON|BSONString} userId
+ */
+const removePermissionsThatUserCanAccess = async (userId) => {
+	try {
+		const { success: finished, filePermissions } = await repo.removeFilePermissionsByUserId(userId);
+
+		const trashBinData = filePermissions.map((fp) => ({
+			scope: 'filePermission',
+			data: fp,
+		}));
+
+		return { finished, trashBinData };
+	} catch (err) {
+		logger.warning('error during tombstone dissolve in file permissions', err);
```

Comment:
logger warning is not logged in production, but the throw an error also not work and log error is a break with our error handling. Maybe asyncErrorLog will work for it. But i am not happy about this.

Reply:
https://github.com/hpi-schul-cloud/schulcloud-server/blob/develop/src/errors/utils.js#L48